apps/www/pages/sla.mdx

```diff
@@ -106,12 +68,12 @@ Business hours are from 6am to 6pm (local time), except where otherwise stated.
 - (vii) That result from your equipment, software, or other technology.
 - (viii) Arising from our suspension and termination of your right to use Supabase in accordance with our Terms.
 
-### Database SLA Exclusions
+#### Database SLA Exclusions
 
 - (i) Unofficially supported Postgres extensions are excluded from our SLA.
```

Comment:
Do we have a documented list we could link to here?

Reply:
@tylerfontaine I believe we do here - https://supabase.com/docs/guides/database/extensions